cluster-autoscaler/go.mod

```diff
@@ -52,9 +52,10 @@ require (
 	k8s.io/cloud-provider-gcp/providers v0.28.2
 	k8s.io/component-base v0.33.0-alpha.0
 	k8s.io/component-helpers v0.33.0-alpha.0
+	k8s.io/dynamic-resource-allocation v0.0.0
 	k8s.io/klog/v2 v2.130.1
 	k8s.io/kubelet v0.33.0-alpha.0
-	k8s.io/kubernetes v1.33.0-alpha.0
+	k8s.io/kubernetes v1.32.0
```

Comment:
Why are the other k8s.io deps still on v0.33.0-alpha.0?

Reply:
Updated the change, as you suggested offline I needed to first revert the dependency update to 1.33.0-alpha.0.